tempesta_fw/t/fuzzer.c

```diff
@@ -33,14 +34,19 @@ MODULE_VERSION("0.1.2");
 MODULE_LICENSE("GPL");
```

Comment:
Please don't forget to update module version for such large changes, `0.1.3` should be now.